test/integration/cmn/template/channel_transfer_template.go

```diff
@@ -0,0 +1,53 @@
+package template
+
+const DefaultChannelTransfer = `{{ with $w := . -}}
+logLevel: debug
+logType: console
+profilePath: {{ .ConnectionPath User }}
+userName: backend
+listenAPI:
+  accessToken: {{ .ChannelTransferAccessToken }}
+  addressHTTP: {{ .ChannelTransferHostAddress }}:{{ .ChannelTransferPort "HttpPort" }}
+  addressGRPC: {{ .ChannelTransferHostAddress }}:{{ .ChannelTransferPort "GrpcPort" }}
+service:
+  address: {{ .ChannelTransferHostAddress }}:{{ .ChannelTransferPort "Listen" }}
+options:
+  batchTxPreimagePrefix: batchTransactions
+  collectorsBufSize: 1
+  executeTimeout: 0s
+  retryExecuteAttempts: 3
+  retryExecuteMaxDelay: 2s
+  retryExecuteDelay: 500ms
+  ttl: {{ .ChannelTransferTTL }}s
```

Comment:
Why don't you make the timeout a string? For example: “10800s”.
In case the time needs to be set less than a second or more difficult to set?